ibis/expr/api.py

```diff
@@ -879,6 +885,9 @@ def read_json(sources: str | Path | Sequence[str | Path], **kwargs: Any) -> ir.T
 def read_parquet(sources: str | Path | Sequence[str | Path], **kwargs: Any) -> ir.Table:
     """Lazily load a parquet file or set of parquet files.
 
+    This function delegates to the `read_parquet` method on the current default
+    backend (DuckDB or `ibis.config.default_backend`)
```

Comment:
```suggestion
    backend (DuckDB or `ibis.config.default_backend`).
```

Reply:
The `kwargs` parameter below (and in the other functions) should accordingly be updated to replace:

```
        DuckDB-specific keyword arguments for the file type.
```

by:

```
        Backend-specific keyword arguments for the file type. For the DuckDB
        backend used by default, please refer to:
```

Reply:
done!  and thanks for the review!